### ISSUE TITLE
Server crashed when a linked storage controller was broken. 

### ISSUE THREAD

**Issue by Hulifoxxie** [P3pp3rF1y/SophisticatedStorage#142]
Setting up my system on our small ATM8 server. Realized I didn't like where my storage controller was. It was already linked to 2 chests. When I broke it the entire server crashed. I've attached the crash report below. 

I can't seem to find a way to un-link chests from the system. Is that a thing? If not, how would I safely go about taking my system apart if I want to rearrange or move? 

[crash-2023-04-01_20.42.17-server.txt](https://github.com/P3pp3rF1y/SophisticatedStorage/files/11130956/crash-2023-04-01_20.42.17-server.txt)

**Comment by P3pp3rF1y** [comment]
Could you share more info on the chests / what you used to break the controller? I tried with 2 chests linked separately to controller and noticed you used ars spell to break it so tried with that as well, but I am unable to recreate. Perhaps it's just something I have fixed as a side effect of changes in latest release, but if you had upgrades in, used a spell that breaks in aoe or something similar that would help me narrow it down. 

To avoid the crash you can unlink the storages using storage tool before breaking the controller though it should just work by breaking the controller first and automatically unlink

**Comment by Hulifoxxie** [comment]
My controller was linked to a diamond and a gold upgraded barrel. I broke the controller with a break projectile spell with pickup added to it. No AOE, just a single block break and collect.  I have tried unlinking it with the tool and I still see all the connecting strings. I should just be able to right-click a linked Storage Link to disconnect it from the system right?

**Comment by P3pp3rF1y** [comment]
yes when it says unlinking from controller on the screen you should be able to do that. Though if you're doing that what you are describing almost sounds like there are two links that exist from that controller to block or somehow there's something like a phantom controller there. Did you try moving that controller somehow with an item from another mod or something similar to that? Even that shouldn't cause a crash in the end, but could give me an idea of how to recreate your case

**Comment by Hulifoxxie** [comment]
I've attached the text I see when flipping between options on the Storage Tool. I've also attached a screenshot showing the mod versions to see if maybe that might be an issue.  I haven't had a further issue, but I haven't tried to move this new controller that I have. The one that crashed the server was the first one I had made, I just realized I didn't like where it was as I started setting up the system. 

Linking to controller at [location]
Showing/Hiding Locks
Showing/Hiding Counts
Locking/Unlocking

I don't have the option to unlink. 

![image](https://user-images.githubusercontent.com/129635851/229622241-01dd9c61-67c8-48ff-b072-113c128d0bb1.png)
![image](https://user-images.githubusercontent.com/129635851/229623181-104691e0-5630-4528-9672-157c1f687adb.png)
